blueprints/showcase-plugin-with-media/readme.md

```diff
@@ -0,0 +1,33 @@
+# Showcase custom plugin from own server with media files and custom theme
```

Comment:
Blueprints in the Gallery usually don't have readmes, we store the description in the Blueprint.

Reply:
I'm wondering what added value could a readme provide and how would users be able to discover it.